readme.txt

```diff
@@ -110,6 +110,15 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.4.0 =
+
+**Release Date:** Februar 17, 2021.
```

Comment:
Typo: *February*

Reply:
Argh, I definitely need to clean my keyboard